src/components/Search/Result.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import 'react-native'
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react-native'
+
+import SearchResult from './Result'
+
+describe( '<SearchResult />', () => {
+  it( 'should handle an onPress event', () => {
```

Comment:
Not descriptive enough, and doesn't describe what the behaviour is

```suggestion
  it( 'given a press, should fire onPress with details of the pressed line', () => {
```